README.md

```diff
@@ -56,6 +56,28 @@ Some syntax errors can be fixed by phpcs.
 composer phpcs:fix
 ```
 
+**WordPress Unit Tests**
+In order to run WordPress unit tests, the test framework needs to be set up.
+```
+/bin/bash /path/to/headless-framework/plugins/wpe-headless/tests/install-wp-tests.sh wpe_headless_tests db_name db_password
+```
```

Comment:
I think it might be helpful here to show the syntax for using socket connections for db-host. I always forget this and have to look it up. Maybe something like this:

```
./tests/install-wp-tests.sh wpe_headless_tests root root localhost:/path/to/mysql/mysqld.sock
```

In Local, mine looks like this:
```
./tests/install-wp-tests.sh wpe_headless_tests root root localhost:/Users/john.parris/Library/ApplicationSupport/Local/run/njgKDZYdQ/mysql/mysqld.sock
```

Reply:
Ah, yes... socket connections. I'll add a comment. Thanks @mindctrl